lib/poteto/review.rb

```diff
@@ -2,27 +2,28 @@
 
 module Poteto
   class Review
-    attr_accessor :path, :line, :comment
+    attr_accessor :path, :line, :comment, :commit_id
     attr_reader :raw_data
 
-    def initialize(raw_data, file_name: nil)
+    def initialize(raw_data, file_name: nil, commit_id: nil)
       @raw_data = raw_data
       @file_name = file_name
       @path = file_name unless file_name.nil?
+      @commit_id = commit_id unless commit_id.nil?
     end
   end
 
   class RubocopReview < Review
```

Comment:
Style/Documentation: Missing top-level documentation comment for `class Poteto::RubocopReview`.

Reply:
Addressed